File: src/WorkVehicle.java
public class WorkVehicle extends Vehicle {
    private double maxCapacity;
    public WorkVehicle(int year, String brand, String model, String spz, double maxCapacity) {
        super(year,brand,model,spz);
        this.maxCapacity = maxCapacity;
    }
    public double getMaxCapacity() {
        return maxCapacity;
    }
    public void setMaxCapacity(double maxCapacity) {
        this.maxCapacity = maxCapacity;
    }

    @Override
    public void displayInfo(){
        super.displayInfo();
        System.out.println("Work vehicle with capacity: " + maxCapacity + "t");
    }

}

File: src/WorkVehicle.kt
class WorkVehicle(year: Int, brand: String, model: String, spz: String, var maxCapacity: Double) :
    Vehicle(year, brand, model, spz) {
    public override fun displayInfo() {
        super.displayInfo()
        println("Work vehicle with capacity: ${maxCapacity}t")
    }

    override fun drive() {
        println("Work vehicle $brand $model $spz can load ${maxCapacity}t")
    }
}